backoffice/backoffice/workflows/api/views.py

```diff
@@ -104,8 +104,8 @@ class AuthorWorkflowViewSet(viewsets.ViewSet):
     serializer_class = WorkflowAuthorSerializer
 
     @extend_schema(
-        summary="Create a New Author",
-        description="Creates a new author, launches the required airflow dags.",
+        summary="Create/Update an Author",
+        description="Creates/Updates an author, launches the required airflow dags.",
         request=serializer_class,
     )
     def create(self, request):
```

Comment:
for updating the author, it’s better to use the update function of the viewset, ‘PUT’

Reply:
Okay as per the current implementation of inspirehep, this endpoint will already work for create and update
https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/submissions/views.py#L153-L174

Creating additional endpoint for update only. Should inspirehep be made to use it?

Reply:
I though we have already a put on hep, let's do it in the next iteration. Could you please create an issue and I will fill in the details thanks

Reply:
Okay thanks, but please do it only when you comeback as vacations are a sacred thing

https://github.com/cern-sis/issues-inspire/issues/528